ui/apps/platform/src/Containers/Violations/Details/Deployment/ContainerConfiguration.tsx

```diff
@@ -74,11 +75,15 @@ function ContainerConfiguration({ container }): ReactElement {
     );
 }
 
-function ContainerConfigurations({ deployment }): ReactElement {
+export type ContainerConfigurationsProps = {
+    deployment: Deployment | null;
+};
+
+function ContainerConfigurations({ deployment }: ContainerConfigurationsProps): ReactElement {
```

Comment:
I didn't know that this was existed, and redid this work in order to fix the Deployment Details for NG 2.0 (the container config had been wrong for years in NG 1.0).

We should look to consolidate these two in a consolidated component, if possible.

ui/apps/platform/src/Components/DeploymentContainerConfig.tsx

Reply:
Good catch, worth tracking in a Jira ticket under the "NG carryover to 4.2" epic?